subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/gradle/ConsumerProvidedTask.java

```diff
@@ -19,12 +19,13 @@
 import com.google.common.collect.Sets;
 import org.gradle.tooling.internal.consumer.converters.TaskNameComparator;
 
+import java.io.Serializable;
 import java.util.SortedSet;
 
 /**
  * A consumer-side implementation of {@link org.gradle.tooling.model.Task}.
  */
-public class ConsumerProvidedTask implements TaskListingLaunchable {
+public class ConsumerProvidedTask implements TaskListingLaunchable, Serializable {
```

Comment:
Testing GH reviews, this time just a clarification:
Why does this now need to be Serializable?